pylabnet/hardware/ni_daqs/nidaqmx_card.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+"""
+This file contains the pylabnet Hardware module class for a generic NI DAQ mx card.
+
+"""
+
+import nidaqmx  
+from pylabnet.utils.logging.logger import LogHandler
+
+
+class Ni_Daq_Mx_Card():
```

Comment:
@cknaut Same style comment here - don't use underscores in class names, use CamelCase.